src/main/java/me/mrCookieSlime/Slimefun/api/BlockStorage.java

```diff
@@ -378,14 +376,21 @@ private static BlockInfoConfig parseBlockInfo(Location l, String json) {
         }
     }
 
-    private static String serializeBlockInfo(Config cfg) {
-        JsonObject json = new JsonObject();
 
-        for (String key : cfg.getKeys()) {
-            json.add(key, new JsonPrimitive(cfg.getString(key)));
+    private static String serializeBlockInfo(Config cfg) {
+        try {
+            StringWriter stringWriter = new StringWriter();
+            JsonWriter jsonWriter = new JsonWriter(stringWriter);
+            jsonWriter.setLenient(true);
+            jsonWriter.beginObject();
+            for (String key : cfg.getKeys()) {
+                jsonWriter.name(key).value(cfg.getString(key));
+            }
+            jsonWriter.endObject();
+            return stringWriter.toString();
+        } catch (IOException e) {
+            throw new AssertionError(e);
```

Comment:
An AssertionError is completely inappropriate here.
No Assertion was made and catching an exception just to throw another one serves no benefit here.
You hsould instead properly log the exception rather than creating a new one that makes no real sense here.